### PR TITLE
Feat api multiple output types

### PR DESCRIPTION
**Pull request by danangmassandy**
TODO:

- [x] Validations
- [x] Convert Tahmo results

### CODE REVIEW

File: django_project/gap/models/measurement.py [hunk 1]
```diff
@@ -40,6 +40,9 @@ class DatasetAttribute(models.Model):
     source_unit = models.ForeignKey(
         Unit, on_delete=models.CASCADE
     )
+    has_ensembles = models.BooleanField(
```
Comment (meomancer) [inline django_project/gap/models/measurement.py:43]: Could you add help text here @danangmassandy ?

File: django_project/gap_api/api_views/measurement.py [hunk 2]
```diff
@@ -153,7 +178,67 @@ def _read_data(self, reader: BaseDatasetReader) -> DatasetReaderValue:
...
+            start_dt: datetime, end_dt: datetime) -> Response:
+        reader:BaseDatasetReader = list(reader_dict.values())[0]
+        reader.read()
+        data_value = reader.get_data_values2()
+        response = StreamingHttpResponse(data_value.to_netcdf_stream(), content_type='application/x-netcdf')
+        response['Content-Disposition'] = 'attachment; filename="data.nc"'
+
+        return response
+
+    def _read_data_as_csv(self, reader_dict: Dict[int, BaseDatasetReader],
```
Comment (meomancer) [inline django_project/gap_api/api_views/measurement.py:221]: I think you could use this renderer
https://github.com/kartoza/tomorrownow_gap/pull/110/files#diff-b33585d0b9d42058de5070e2fa350a33141e18912540a41f01644f3d5c0db1d6R17

File: django_project/gap_api/api_views/measurement.py [hunk 3]
```diff
@@ -153,7 +178,67 @@ def _read_data(self, reader: BaseDatasetReader) -> DatasetReaderValue:
...
+            values = reader.get_data_values2().to_json()
+            if values:
+                data['metadata']['dataset'].append({
+                    'provider': reader.dataset.provider.name,
+                    'attributes': reader.get_attributes_metadata()
+                })
+                data['results'].append(values)
+        return data
+
+    def _read_data_as_netcdf(
```
Comment (meomancer) [inline django_project/gap_api/api_views/measurement.py:210]: Maybe we can create new renderer

File: django_project/gap_api/api_views/measurement.py (hunk 2 above)
Comment (danangmassandy) [reply django_project/gap_api/api_views/measurement.py:221]: yeah, I'll refactor to use renderer like in crop plan API

**Pull request merged**